datastore/models.go

```diff
@@ -441,13 +442,13 @@ type Event struct {
 	// with your internal systems.
 	// This is optional
 	// If not provided, we will generate one for you
-	ProviderID string                `json:"provider_id,omitempty" bson:"provider_id"`
-	SourceID   string                `json:"source_id,omitempty" bson:"source_id"`
-	GroupID    string                `json:"group_id,omitempty" bson:"group_id"`
-	EndpointID string                `json:"endpoint_id,omitempty" bson:"endpoint_id"`
-	Headers    httpheader.HTTPHeader `json:"headers" bson:"headers"`
-	Endpoint   *Endpoint             `json:"endpoint_metadata,omitempty" bson:"-"`
-	Source     *Source               `json:"source_metadata,omitempty" bson:"-"`
+	ProviderID       string                `json:"provider_id,omitempty" bson:"provider_id"`
+	SourceID         string                `json:"source_id,omitempty" bson:"source_id"`
+	GroupID          string                `json:"group_id,omitempty" bson:"group_id"`
+	Endpoints        []string              `json:"endpoints" bson:"endpoints"`
```

Comment:
the api might not be backward compatible if we do this

Reply:
Yeah I plan on addressing this in the backward compatible PR

Reply:
ok